transport/http/outbound.go

```diff
@@ -386,6 +390,9 @@ func (o *Outbound) withCoreHeaders(req *http.Request, treq *transport.Request, t
 		req.Header.Set(EncodingHeader, encoding)
 	}
 
+	// TODO(pedge): update to true when this is supported
+	req.Header.Set(AcceptResponseErrorHeader, AcceptFalse)
```

Comment:
Or omit until it is, but this is good signaling for where you intend to.

Reply:
commented out now that only `AcceptTrue` is defined